server/index.js

```diff
@@ -0,0 +1,56 @@
+const express = require('express');
+const db = require('../db/index.js');
+const bodyParser = require('body-parser');
+// for cross-origin-request
+const cors = require('cors');
+const path = require('path');
+
+let app = express();
+app.use(bodyParser.json());
+app.use(cors());
+
+//server static content
+app.use(express.static(path.join(__dirname, '../client/dist')))
+app.get('/reviews', function(req, res) {
+    var pageIndex = Number(req.url.split('=')[1]);
+    var getFive; var len;   
+    var endIndex = pageIndex + 5;
+    var startIndex = pageIndex;
+    var reviewInfo; 
+    console.log('inside app get method-', req.url, reviewInfo)
+    //if (pageIndex === 0) {
+        db.find(function(err, results) {
```

Comment:
Wouldn't you need to find all the reviews for a specific room/listing id? Instead of returning all the reviews for all rooms?

Reply:
I was checking pagination, since i limited number of reviews to 10(for test purpose) , I was trying to render everything in the server..so that I could work on pagination